core/templates/pages/contributor-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -127,8 +127,9 @@ export class ContributionOpportunitiesBackendApiService {
     });
   }
 
-  fetchTranslationOpportunities(languageCode: string, cursor: string):
-  Promise<TranslationContributionOpportunities> {
+  async fetchTranslationOpportunitiesAsync(
```

Comment:
@KhushiGangopadhyay, have you tested the changes? It looks like the e2e tests are failing for the contributor dashboard page.